index.html

```diff
@@ -0,0 +1,173 @@
+          </ul>
+        </div>
+      </div>
+      <nav class="nav-bar">
+        <div class="container">
+          <div class="logo">
+            <a href="index.html"><img
+              src="assets/images/yaycommerceNobg.png"
+              alt="logo"
+              class="logo"
+            /></a>
+          </div>
+          <ul class="navigation">
+            <li><a href="index.html" class="home">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#" class="contact">Buy Now</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div class="toggle"></div>
+      </div>
+      <div class="container">
+        <section class="main-title">
+          <h1><q>Wellcome to YayCommerce</q></h1>
+          <h2>
+            Ethiopia's Largest <br />
+            E-commerce Site
+          </h2>
+        </section>
+        <div class="main-description">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto
+            quidem consequatur laboriosam accusantium a eveniet accusamus dicta
+          </p>
+        </div>
+        <section class="main-address">
+          <h3>Addis Ababa, Ethiopia</h3>
+          <p>@&nbsp;Akaki Kality sub city, Wereda 07, Kebele 02, Youth Hall</p>
+        </section>
+      </div>
+    </main>
+    <section class="main-services">
+      <h3 class="title">Main Services</h3>
+      <div class="container">
+        <div class="services">
+          <div class="service-1">
+            <img src="assets/images/ecommerce.png" alt="Ecommerce icon" />
+            <h3>Cosmotics</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="service-2">
+            <img src="assets/images/shopping(1).png" alt="shopping icon" />
+            <h3>Watches</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="service-3">
+            <img src="assets/images/warehouse.png" alt="warehouse icon" />
+            <h3>Electronics</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="service-4">
+            <img src="assets/images/shopping.png" alt="shopping icon" />
+            <h3>Shoes</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="service-5">
+            <img src="assets/images/shopping(1).png" alt="shopping icon" />
+            <h3>T-shirt</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+        </div>
+        <button class="see-whole-products" type="button">See The Whole Products</button>
```

Comment:
The mobile version of the button isn't responsive. The font size is too big. Kindly reduce the font size that work for both Desktop and Mobile. 😃 

<img width="293" alt="Screen Shot 2022-04-15 at 5 10 09 AM" src="https://user-images.githubusercontent.com/83048208/163495857-7fc5ef53-f380-4d70-85e2-776e6c669162.png">